### PR TITLE
test/yamitranscode: fix decode output file format is always I420.

### PR DESCRIPTION
**Pull request by feiwan1**
Use specific fourcc format or format resolved from output file
to create vpp output in yamitranscode.

Signed-off-by: Wang,Fei <fei.w.wang@intel.com>

### CODE REVIEW

File: tests/yamitranscode.cpp [hunk 1]
```diff
@@ -395,7 +395,7 @@ class TranscodeTest
             return false;
         }
         m_input = createInput(m_cmdParam, m_display);
-        m_output = createOutput(m_cmdParam, m_display, m_input->getFourcc());
+        m_output = createOutput(m_cmdParam, m_display, m_cmdParam.fourcc);
```
Comment (uartie) [inline tests/yamitranscode.cpp]: m_cmdParam.fourcc option is a little confusing... it is applied to VppInput too.  Should input and output have their own fourcc cmdline options?  How would user transcode NV12 input to YV12 output?

**Comment by uartie** [comment]
What use-case are you trying to fix?  Can you give an example?

File: tests/yamitranscode.cpp (hunk 1 above)
Comment (feiwan1) [reply tests/yamitranscode.cpp]: @uartie , yamitranscode can get the output fourcc by the output filename. if without this patch, when I use 'yamitranscode -i 1920x1080.NV12 1920x1080.YV12' want to convert NV12 to YV12. But the output file 1920x1080.YV12 actually is still NV12 format.

**Comment by uartie** [comment]
please rebase to fix CI builds

File: tests/yamitranscode.cpp (hunk 1 above)
Comment (xuguangxin) [reply tests/yamitranscode.cpp]: @feiwan1 , it's not right, we need m_input->getFourcc() to tell output, we need 10 bits encoder.
maybe we can remvoe -s command in yamitranscode, it's not even listed in help.
and we may consider removing VppOutputFile branch in createOutput. 
The transcode output is raw frame does not make sense.

**Comment by uartie** [comment]
Closing as inactive... please resubmit if you feel this is still relevant.

**Pull request closed without merging**